packages/@okta/vuepress-theme-prose/package.json

```diff
@@ -27,15 +27,13 @@
     "@okta/okta-signin-widget": "^5.4.0",
     "axios": "^0.21.0",
     "cookies.js": "^2.1.15",
-    "copy-webpack-plugin": "^8.1.0",
     "css-to-json": "^2.1.0",
     "include-media": "^1.4.9",
     "jshint": "^2.12.0",
     "jwt-decode": "^3.1.2",
     "moment": "^2.24.0",
     "moment-timezone": "^0.5.32",
     "query-string": "^6.13.8",
-    "sass": "^1.32.7",
```

Comment:
Not sure this was ever used but I see no imports, and removing it has no effects.